### PR TITLE
US2D get path between two locations

### PR DESCRIPTION
**Pull request by TalalBazerbachi**

### CODE REVIEW

File: components/home/index.js [hunk 1]
```diff
@@ -34,27 +36,37 @@ class Home extends Component {
...
       }
     });
-  }
+  };
 
-  // Function : change the visiblity of the switchCampuses component
-  // parameter: boolean to set the visibility (false: unvisible)
-  changeVisibilityTo = (visibility) => {
-    this.setState({ isVisible: visibility });
+  changeVisibilityTo = boolean => {
```
Comment (gisellemartel) [inline components/home/index.js]: boolean is a reserved keyword, please use an appropriate variable name

File: components/home/index.js [hunk 2]
```diff
@@ -34,27 +36,37 @@ class Home extends Component {
...
+    this.setState({ isVisible: boolean });
+  };
+  changeVisibilityToSearch = boolean => {
+    this.setState({ isVisible2: boolean})
   }
 
-  // Function: Updates coordinates state to draw polyline
-  // Parameter: object with latitudes and longitudes
-  updateCoordinates = (newCoordinates) => {
+  updateRegion2 = newRegion2 => {
```
Comment (gisellemartel) [inline components/home/index.js:49]: Why do you have a second function with the same name? Is this necessary

File: components/home/index.js [hunk 3]
```diff
@@ -34,27 +36,37 @@ class Home extends Component {
...
 
-  // Function : change the visiblity of the switchCampuses component
-  // parameter: boolean to set the visibility (false: unvisible)
-  changeVisibilityTo = (visibility) => {
-    this.setState({ isVisible: visibility });
+  changeVisibilityTo = boolean => {
+    this.setState({ isVisible: boolean });
+  };
+  changeVisibilityToSearch = boolean => {
+    this.setState({ isVisible2: boolean})
```
Comment (gisellemartel) [inline components/home/index.js]: instead of calling it isVisible2, why not have isSearchVisibility. This style of naming convention is a little confusing

**Review by gisellemartel — CHANGES_REQUESTED** [review]

File: components/home/index.js (hunk 2 above)
Comment (TalalBazerbachi) [reply components/home/index.js:49]: this function is used to update the region in the second search bar.

File: components/home/index.js [hunk 4]
```diff
@@ -34,27 +36,38 @@ class Home extends Component {
         longitudeDelta: 0.05
       }
     });
-  }
+  };
 
-  // Function : change the visiblity of the switchCampuses component
-  // parameter: boolean to set the visibility (false: unvisible)
-  changeVisibilityTo = (visibility) => {
```
Comment (gisellemartel) [inline components/home/index.js:41]: make sure not to delete the comments that explain the code

File: components/home/index.js [hunk 5]
```diff
@@ -34,27 +36,38 @@ class Home extends Component {
...
+        latitude: newRegion2.latitude,
+        longitude: newRegion2.longitude,
+        latitudeDelta: 0.05,
+        longitudeDelta: 0.05
+      }
+    });
+  };
 
-  // Function: Updates coordinates state to draw polyline
-  // Parameter: object with latitudes and longitudes
```
Comment (gisellemartel) [inline components/home/index.js:46]: Please re-add this comment

**Review by gisellemartel — CHANGES_REQUESTED** [review]

**Pull request closed without merging**